library/common/config/config.cc

```diff
@@ -454,6 +457,9 @@ stats_sinks: *stats_sinks
         - safe_regex:
             google_re2: {}
             regex: '^http.hcm.downstream_rq_[\w]+'
+        - safe_regex:
+            google_re2: {}
+            regex: '.*pbf_filter.*'
```

Comment:
can we make this regex more specific? Can we specify the prefix as opposed to using `.*` matching?